plugin/trino-hive/src/test/java/io/trino/plugin/hive/AbstractTestHiveLocal.java

```diff
@@ -126,4 +159,129 @@ public void testGetTableSchemaOffline()
     {
         throw new SkipException("Test disabled for this subclass");
     }
+
+    @Test
+    public void testSparkBucketedTableValidation()
+            throws Exception
+    {
+        SchemaTableName table = temporaryTable("spark_bucket_validation");
+        try {
+            doTestSparkBucketedTableValidation(table);
+        }
+        finally {
+            dropTable(table);
+        }
+    }
+
+    private void doTestSparkBucketedTableValidation(SchemaTableName tableName)
+            throws Exception
+    {
+        java.nio.file.Path externalLocation = copyResourceDirToTemporaryDirectory("spark_bucketed_nation");
+        try {
+            createExternalTable(
+                    tableName,
+                    ORC,
+                    ImmutableList.of(
+                            new Column("nationkey", HIVE_INT, Optional.empty()),
+                            new Column("name", HIVE_STRING, Optional.empty()),
+                            new Column("regionkey", HIVE_INT, Optional.empty()),
+                            new Column("comment", HIVE_STRING, Optional.empty())),
+                    ImmutableList.of(),
+                    Optional.of(new HiveBucketProperty(
+                            ImmutableList.of("nationkey"),
+                            BUCKETING_V1,
+                            3,
+                            ImmutableList.of(new SortingColumn("name", SortingColumn.Order.ASCENDING)))),
+                    new Path(URI.create("file://" + externalLocation.toString())));
```

Comment:
what happens if you replace this with 
```
new Path(Resources.getResource("spark_bucketed_nation").toURI())
```

?

(using `com.google.common.io.Resources`)

Reply:
Hm.... weird. We load resouces from src/test/resources as _files_ relatively often in tests

maybe it was iniitally in a different module?

Reply:
Oh, the test is actually run from `trino-hive-hadoop2`, right?

Reply:
I see now we this will fail. You can revert `Use resource directory directly instead of copying to tempDir` commit, no need for CI.